### ISSUE TITLE
Failed to build the docker image.

### ISSUE THREAD

**Issue by cryptcoin-junkey** [CounterpartyXCP/counterwallet#868]
`docker build .` on the develop branch.

```
Removing intermediate container 10eff24d760e
Step 42/48 : RUN grunt build
 ---> Running in 9502ddd77370
Running "build:process" (build) task
Generate jQuery-TimeAgo localizations
Warning: Unable to read "src/vendors/jquery-timeago/locales/jquery.timeago.en.js" file (Error code: ENOENT). Use --force to continue.

Aborted due to warnings.
The command '/bin/sh -c grunt build' returned a non-zero code: 3
```

**Comment by sgirad** [comment]
I have this problem.
What is the solution?
Please help me.

**Comment by jdogresorg** [comment]
use fednode to install counterwallet and it should work fine. Also make sure you actually need to run counterwallet, in most cases `fednode install base master` where you just install bitcoin/addrindexrs/counterparty is what you need to access counterparty, you don't need to install counterwallet unless you want to run a counterwallet instance.

**Comment by sgirad** [comment]
Thank you for your help.
It finally installed from `fednode install full master` and I wanna start at localhost.
where can I find this guide?
I would like to know about changing the config at localhost. 
Counterwallet was started at localhost but found an error stating that the server could not be found.
And `fednode tail counterblock` is with this message
`counterblock_1           | WARNING:blockfeed: Cannot contact counterparty-server (via get_running_info): 'NoneType' object is not subscriptable
counterblock_1           | WARNING:util: Got call_jsonrpc_api request error: result is None -- is the 'http://counterparty:4000/api/' endpoint operational?`
what is this mean?

**Comment by jdogresorg** [comment]
fednode has to spend some time getting things synced up. first it has to sync bitcoin, then addrindexrs, then counterparty.. the above counterblock messages is saying that counterblock can't talk to counterparty.. which is probably because things are not synced up yet

Make sure bitcoin is synced via `fednode tail bitcoin`

please review the fednode install guide, as this information is covered there https://counterparty.io/docs/federated_node/

**Comment by sgirad** [comment]
Thanks for your reply.
I ran `fednode tail bitcoin`
Then I got these code.
Bitcoin isn't synced yet?
---------------------------------------------
bitcoin_1                | 2021-08-25T15:33:43Z Checking all blk files are present...
bitcoin_1                | 2021-08-25T15:33:43Z Opening LevelDB in /root/.bitcoin/chainstate
bitcoin_1                | 2021-08-25T15:33:43Z Opened LevelDB successfully
bitcoin_1                | 2021-08-25T15:33:43Z Using obfuscation key for /root/.bitcoin/chainstate: b9d7b7f3b9cb9ea7
bitcoin_1                | 2021-08-25T15:33:43Z Loaded best chain: hashBestChain=000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f height=0 date=2009-01-03T18:15:05Z progress=0.000000
bitcoin_1                | 2021-08-25T15:33:43Z init message: Rewinding blocks...
bitcoin_1                | 2021-08-25T15:33:43Z FlushStateToDisk: write coins cache to disk (0 coins, 0kB) started
bitcoin_1                | 2021-08-25T15:33:43Z FlushStateToDisk: write coins cache to disk (0 coins, 0kB) completed (0.00s)
bitcoin_1                | 2021-08-25T15:33:43Z init message: Verifying blocks...
bitcoin_1                | 2021-08-25T15:33:43Z  block index              41ms
bitcoin_1                | 2021-08-25T15:33:43Z Opening LevelDB in /root/.bitcoin/indexes/txindex
bitcoin_1                | 2021-08-25T15:33:43Z Opened LevelDB successfully
bitcoin_1                | 2021-08-25T15:33:43Z Using obfuscation key for /root/.bitcoin/indexes/txindex: 0000000000000000
bitcoin_1                | 2021-08-25T15:33:43Z block tree size = 1
bitcoin_1                | 2021-08-25T15:33:43Z nBestHeight = 0
bitcoin_1                | 2021-08-25T15:33:43Z txindex thread start
bitcoin_1                | 2021-08-25T15:33:43Z txindex is enabled at height 0
bitcoin_1                | 2021-08-25T15:33:43Z txindex thread exit
bitcoin_1                | 2021-08-25T15:33:43Z loadblk thread start
bitcoin_1                | 2021-08-25T15:33:43Z Bound to [::]:8333
bitcoin_1                | 2021-08-25T15:33:43Z Bound to 0.0.0.0:8333
bitcoin_1                | 2021-08-25T15:33:43Z Bound to 127.0.0.1:8334
bitcoin_1                | 2021-08-25T15:33:43Z init message: Loading P2P addresses...
bitcoin_1                | 2021-08-25T15:33:43Z Imported mempool transactions from disk: 0 succeeded, 0 failed, 0 expired, 0 already there, 0 waiting for initial broadcast
bitcoin_1                | 2021-08-25T15:33:43Z loadblk thread exit
bitcoin_1                | 2021-08-25T15:33:43Z torcontrol thread start
bitcoin_1                | 2021-08-25T15:33:43Z Loaded 702 addresses from peers.dat  11ms
bitcoin_1                | 2021-08-25T15:33:43Z Loaded 0 addresses from "anchors.dat"
bitcoin_1                | 2021-08-25T15:33:43Z 0 block-relay-only anchors will be tried for connections.
bitcoin_1                | 2021-08-25T15:33:43Z init message: Starting network threads...
bitcoin_1                | 2021-08-25T15:33:43Z net thread start
bitcoin_1                | 2021-08-25T15:33:43Z dnsseed thread start
bitcoin_1                | 2021-08-25T15:33:43Z Waiting 11 seconds before querying DNS seeds.
bitcoin_1                | 2021-08-25T15:33:43Z opencon thread start
bitcoin_1                | 2021-08-25T15:33:43Z msghand thread start
bitcoin_1                | 2021-08-25T15:33:43Z addcon thread start
bitcoin_1                | 2021-08-25T15:33:43Z init message: Done loading
bitcoin_1                | 2021-08-25T15:33:54Z Loading addresses from DNS seed seed.bitcoin.sprovoost.nl
bitcoin_1                | 2021-08-25T15:33:56Z Loading addresses from DNS seed seed.bitcoinstats.com
bitcoin_1                | 2021-08-25T15:33:57Z Loading addresses from DNS seed dnsseed.bitcoin.dashjr.org
bitcoin_1                | 2021-08-25T15:34:07Z Waiting 11 seconds before querying DNS seeds.
bitcoin_1                | 2021-08-25T15:34:18Z Loading addresses from DNS seed seed.bitcoin.jonasschnelli.ch
bitcoin_1                | 2021-08-25T15:34:18Z Loading addresses from DNS seed seed.btc.petertodd.org
bitcoin_1                | 2021-08-25T15:34:20Z Loading addresses from DNS seed seed.bitcoin.wiz.biz
bitcoin_1                | 2021-08-25T15:34:25Z Waiting 11 seconds before querying DNS seeds.
bitcoin_1                | 2021-08-25T15:34:36Z Loading addresses from DNS seed seed.bitcoin.sipa.be
bitcoin_1                | 2021-08-25T15:34:37Z Loading addresses from DNS seed dnsseed.emzy.de
bitcoin_1                | 2021-08-25T15:34:39Z Loading addresses from DNS seed dnsseed.bluematt.me
bitcoin_1                | 2021-08-25T15:34:39Z 186 addresses found from DNS seeds
bitcoin_1                | 2021-08-25T15:34:39Z dnsseed thread exit
---------------------------------------------
